openpype/hosts/houdini/plugins/publish/validate_unreal_staticmesh_naming.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+"""Validator for correct naming of Static Meshes."""
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from openpype.pipeline.publish import ValidateContentsOrder
+
+from openpype.hosts.houdini.api.action import SelectInvalidAction
+from openpype.hosts.houdini.api.lib import get_output_children
+
+import hou
+
+
+class ValidateUnrealStaticMeshName(pyblish.api.InstancePlugin,
+                                   OptionalPyblishPluginMixin):
+    """Validate name of Unreal Static Mesh.
+
+    This validator checks if output node name has a collision prefix:
+            - UBX
+            - UCP
+            - USP
+            - UCX
+
+    This validator also checks if subset name is correct
+            - {static mesh prefix}_{Asset-Name}{Variant}.
```

Comment:
I feel like this is mixing two validations together that should be separate. You now can only disable both of these at the same time even though they are quite different things - right.

Also, out of curiosity - when would the subset name be incorrect?

Reply:
tbh, I was imitating the corresponding maya validator which do the two things! 
but yeah, why not splitting them into two! 

> Also, out of curiosity - when would the subset name be incorrect?

as far as I tested, it only happens when subset is changed manually. 
![image](https://github.com/ynput/OpenPype/assets/20871534/ef048f56-1992-4d88-a7ae-78f032fb51ed)